scan/src/components/oracle-script/OracleScriptBridgeCode.re

```diff
@@ -141,45 +140,38 @@ module LanguageIcon = {
         src={
           switch (icon) {
           | Solidity => Images.solidityIcon
-          | Vyper => Images.vyperIcon
+          /*           | Vyper => Images.vyperIcon */
           | Go => Images.golangIcon
-          | PACT => Images.pactIcon
+          /*           | PACT => Images.pactIcon */
           }
         }
       />
     </div>;
   };
 };
 
-[@react.component]
-let make = () => {
-  let description = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent aliquet tempor imperdiet. Morbi tincidunt molestie tortor a finibus. Nulla hendrerit iaculis metus, in laoreet tellus eleifend vel. Aliquam pretium porta mi, a efficitur justo ullamcorper sed. Donec interdum accumsan nibh, sed tempor lectus rutrum ac. Morbi et magna in magna varius iaculis. Praesent mollis nulla non arcu ullamcorper, at bibendum nibh pellentesque. Aenean ac quam eget turpis euismod lacinia. Phasellus libero lectus, pulvinar at ipsum non, ullamcorper commodo felis.";
-  let codetest = {j|
-    pragma solidity ^0.5.0;
-
-    import "./Borsch.sol";
-
-    library ResultDecoder {
-      using Borsh for Borsh.Data;
-
-      struct Result {
-        string symbol;
-        uint64 multiplier;
-        uint8 what;
-      }
+let getFileNameFromLanguage = (~language) => {
+  switch (language) {
+  | Solidity => "ResultDecoder.sol"
+  | Go => "ResultDecoder.go"
+  };
+};
 
-      function decodeResult(bytes memory _data)
-        internal
-        pure
-        returns (Result memory result)
-      {
-          Borsh.Data memory data = Borsh.from(_data);
-          result.symbol = string(data.decodeBytes());
-          result.multiplier = data.decodeU64();
-          result.what = data.decodeU8();
-      }
-    }|j};
+let getCodeFromSchema = (~schemaOpt, ~language) => {
+  {
+    let%Opt schema = schemaOpt;
+    let code =
+      switch (language) {
+      | Solidity => Borsh.generateSolidity(schema, "Output") |> Belt_Option.getWithDefault(_, "")
+      | Go => Borsh.generateGo("main", schema, "Output") |> Belt_Option.getWithDefault(_, "")
+      };
+    Some(code);
+  }
+  |> Belt_Option.getWithDefault(_, "");
+};
```

Comment:
Let's return `codeOpt`, then handle option case on the render.
If code is available, you render it the same way.
If not, you just render the message "Code is not verified."